src/components/ListView/components/Item/Item.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Hover, Wrapper } from './components';
+import { sizes } from '../../../../styles/constants';
+
+const { KILO, MEGA, GIGA } = sizes;
+
+const Item = ({ children, ...props }) => (
+  <Wrapper {...props}>
+    {children}
+    <Hover />
```

Comment:
One quick question: Why not providing the hover styles as a css declaration and use it on the Wrapper instead of rendering a component for it?

Reply:
This component provides me with an extra border that I can work on without changing the original element in any way. It is also much cleaner solution css wise.